src/hooks/useSession.js

```diff
@@ -1,6 +1,8 @@
 import useSWR, { mutate } from "swr";
 
-import { API_HOSTNAME } from "../config";
+import config from "../config";
+const { API_HOSTNAME } = config;
+
```

Comment:
It's a React hook that components can use to get the current user session. If a user is logged in, it will return data for that user, such as email, company details (if applicable), etc..